package.json

```diff
@@ -8,7 +8,7 @@
     "postinstall": "esno scripts/prepare.ts",
     "taze": "taze minor -wIr && pnpm -r --parallel run update-post",
     "bench": "npm -C bench run bench",
-    "build": "rimraf packages/*/dist && esno scripts/copy-files.ts && pnpm -r --filter=./packages/* run build && pnpm -r run build-post",
```

Comment:
Because the `@unocss/inspector` is directly import `preset-mini`, we need to build the utils first.
https://github.com/unocss/unocss/blob/02b06c28d6748e6bb6c969597f25bba112238b0f/packages/inspector/src/analyzer.ts#L2

Reply:
We can import the source file from the utils package, so build is not required.

Reply:
The `parseColor` is not in the utils. Since it relies on many `preset-mini` methods. So I think it should stay where it is

Reply:
Then you will need to update the alias.ts and tsconfig.json to point the utils package to the ts entry